templates/home/index.html.twig

```diff
@@ -1,7 +1,61 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}Projektai{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
+
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for project in data %}
+            {% for student in project.students %}
+                    <li class="list-group-item">
+                        <a href="{{ path('student') }}?name={{ student }}&amp;project={{ project.name|replace({' ': ''})|lower }}"> {{ student }}</a>
```

Comment:
Šaunu, kad naudoji Twig filtrus (`replace`, `lower`), bet praktikoje patogu yra perduoti adreso parametrus su `path` funkcija. Pvz.:

```twig
{{ path('student', {'name': name, 'otherParameter': other.data }) }}
```

Plačiau:
  * https://symfony.com/doc/current/templating.html#linking-to-pages
  * https://symfony.com/doc/current/reference/twig_reference.html#path